### PR TITLE
feat: Vision switch functionality is provided on OpenRouter

### PR DESCRIPTION
**Pull request by jeessy2**
# Description

Vision switch functionality is provided on OpenRouter. Copy from openai_api_compatible

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update, included: [Dify Document](https://github.com/langgenius/dify-docs)
- [ ] Improvement, including but not limited to code refactoring, performance optimization, and UI/UX improvement
- [ ] Dependency upgrade

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] TODO

# Suggested Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I ran `dev/reformat`(backend) and `cd web && npx lint-staged`(frontend) to appease the lint gods
- [ ] `optional` I have made corresponding changes to the documentation 
- [ ] `optional` I have added tests that prove my fix is effective or that my feature works
- [ ] `optional` New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by crazywoola — APPROVED** [review]
See this it should be 'radio'

https://github.com/langgenius/dify/blob/5b447d61a6ac1c0d4b0499623a5babfe6f8e5c7e/api/core/model_runtime/model_providers/ollama/ollama.yaml#L88

**Review by crazywoola — CHANGES_REQUESTED** [review]
See this it should be 'radio'

https://github.com/langgenius/dify/blob/5b447d61a6ac1c0d4b0499623a5babfe6f8e5c7e/api/core/model_runtime/model_providers/ollama/ollama.yaml#L88

**Comment by crazywoola** [comment]
> Should I modify the `vision` in `openai_api_compatible.yaml` to `radio`?

Not for now, at least not in this pr.

File: api/core/model_runtime/model_providers/openrouter/openrouter.yaml [hunk 1]
```diff
@@ -73,3 +73,22 @@ model_credential_schema:
...
       default: "4096"
       type: text-input
+    - variable: vision_support
+      show_on:
+        - variable: __model_type
+          value: llm
+      label:
+        zh_Hans: Vision 支持
+        en_US: Vision Support
+      type: radio
```
Comment (crazywoola) [inline api/core/model_runtime/model_providers/openrouter/openrouter.yaml:83]:
```
      type: radio
      required: false
      options:
        - value: 'true'
          label:
            en_US: 'Yes'
            zh_Hans: 是
        - value: 'false'
          label:
            en_US: 'No'
            zh_Hans: 否
```

**Review by crazywoola — CHANGES_REQUESTED** [review]

File: api/core/model_runtime/model_providers/openrouter/openrouter.yaml (hunk 1 above)
Comment (crazywoola) [reply api/core/model_runtime/model_providers/openrouter/openrouter.yaml:83]: The rest should be like this.

Comment (jeessy2) [reply api/core/model_runtime/model_providers/openrouter/openrouter.yaml:83]: It won't be compatible with `openai_api_compatible` If modify the value to `true`, What to do next?
[        return super().get_customizable_model_schema(model, cred_with_endpoint)](https://github.com/langgenius/dify/blob/b9b28900b1dedf4b684dc875cc226695518354fc/api/core/model_runtime/model_providers/openrouter/llm/llm.py#L40)

Comment (crazywoola) [reply api/core/model_runtime/model_providers/openrouter/openrouter.yaml:83]:
```
type: radio
required: false
options:
  - value: 'support'
    label:
      en_US: 'Yes'
      zh_Hans: 是
  - value: 'no_support'
    label:
      en_US: 'No'
      zh_Hans: 否
```

**Review by crazywoola — APPROVED** [review]
LGTM

**Pull request merged**